puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/gemset_cleanup.sh

```diff
@@ -4,5 +4,6 @@
 . /etc/profile.d/rvm.sh
 gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
 rvm use ruby-${ruby}@${gemset}
+bundle exec rake db:drop
```

Comment:
In the plugin job you'll also need to `cd foreman/` as IIRC each different script executes from the job workspace rather than the PWD of the last script.